joplin/settings.py

```diff
@@ -98,6 +98,7 @@
     'pages.information_page',
     'pages.location_page',
     'pages.official_documents_page',
+    'pages.official_documents_collection',
```

Comment:
Let's make the names the same as the model. So a singular "official_document_collection" and "official_document_page"

Reply:
this may be a little thorny. I propose that I lump this in with removing the old OfficialDocumentPage model, handle it on that branch & PR. 

My concern is that renaming an app while also trying to do data migrations may be weird. Maybe I'm being overly cautious though?

Reply:
because I'd have to rename both collection and the pages.official_document_page *everywhere*

Reply:
Good plan, that works